### DISCUSSION TITLE
Test Report looks different when mocha report is merged / not merged

### DISCUSSION THREAD

**Discussion by nischaytv** [test-results-reporter/reporter#74] [Q&A]
When my mocha report (.json file) is merged with all the test results, i get alert in team like this

![image](https://user-images.githubusercontent.com/20622111/183478283-03e7429a-5df3-4250-bf71-707bf18c6d90.png)

But when my mocha report is not merged i get alert with more details like this

![image](https://user-images.githubusercontent.com/20622111/183478400-95e58cf9-3c3d-4619-b263-ee71de358295.png)

Is there anyway i can get alert with full details even after merging the mocha reports?
As of now i'm using cypress-mochawesome-reporter where i can get the merged mocha (.json) file by enabling the flag saveJson: true in config file.

**Comment by ASaiAnudeep** [comment]
Hello @nischaytv , Can you share us these merged and non-merged json files to further investigate the issue.

**Reply by nischaytv** [reply]
Hi @ASaiAnudeep
Here is a folder contains both Single Mocha Reports and Merged Mocha report.
[https://we.tl/t-PM74VBOWfd](https://we.tl/t-PM74VBOWfd)

and below is the code which i have used:

"results":
[
{
"type": "mocha",
"files": ["cypress/reports/*.json"]
}
]

This is how reports looks like
Merged:
![image](https://user-images.githubusercontent.com/20622111/183832648-0265aa21-1c06-4334-9882-b7f087d11910.png)

Single:
![image](https://user-images.githubusercontent.com/20622111/183832685-d11c6d05-f945-4844-abbe-9ec0cdaffe7e.png)


Hope this helps for your investigation.
